reflectable/lib/src/builder_implementation.dart

```diff
@@ -5005,7 +5005,18 @@ Future<String> _extractMetadataCode(Element element, Resolver resolver,
       if (_isPrivateName(name)) {
         await _severe('Cannot access private name $name', element);
       }
-      metadataParts.add('const $prefix$name($arguments)');
+      if (annotationNode.typeArguments != null) {
+        String typeArguments = annotationNode.typeArguments!.arguments
+            .map((TypeAnnotation typeArgument) {
+              LibraryElement library = typeArgument.type!.element!.library!;
```

Comment:
We can't assume that all those getters return a non-null value. In particular, a small test that I just wrote makes the code generator crash in this line.

```
Null check operator used on a null value
package:reflectable/src/builder_implementation.dart 5011:71  _extractMetadataCode.<fn>
```

It looks like it is `arguments` (which ends at column 71).

Reply:
Added some checks for `typeArgument.type` and `typeArgument.type.element`, and that seemed to fix the error